pkgs/development/tools/lazygit/default.nix

```diff
@@ -15,6 +11,11 @@ buildGoPackage rec {
     sha256 = "1jq093nsfh7xqvsjvaad9wvqd3rjrpyp5fl8qxwbhaj3sxx19v7g";
   };
 
+  vendorSha256 = null;
```

Comment:
Sorry, I don't know about Go packagin in NixOS, but is this OK?

Reply:
Should be, the lazygit project has a `vendor` directory so according to [this comment](https://github.com/NixOS/nixpkgs/blob/93fd4b5c6c7875188390da8a336cca41d55ecb06/pkgs/development/go-modules/generic/default.nix#L18) it should be OK

Reply:
Okay then!